src/Menu/index.scss

```diff
@@ -92,3 +92,4 @@
     position: relative;
   }
 }
+
```

Comment:
nit: is this extra blank line necessary?

Reply:
Thanks for catching that! It's now gone.